docs/plugins/python3_migration.rst

```diff
@@ -242,6 +242,30 @@ future import:
 
    from __future__ import absolute_imports
 
+Explicit imports
+................
+
+Sometimes it is necessary to use an import statement that is explicitly related to a specific python version.
+You can do this in differnt ways:
```

Comment:
There are 2 errors:
* "python" should be "Python"
* "differnt" should be "different"